.pipelines/templates/e2e-template.yaml

```diff
@@ -0,0 +1,51 @@
+jobs:
+  - job: E2E_Tests
+    pool:
+      name: nodesigtest-pool
+    timeoutInMinutes: 90
+    displayName: Run AgentBaker E2E
+    variables:
+      - group: ab-e2e
```

Comment:
which variables are stored in that group ?

Reply:
Azure Subscription ID, Resource group etc. Things to create test clusters and download SIG images.